dmoj/executors/compiled_executor.py

```diff
@@ -199,6 +197,42 @@ def on_return():
 
         return self.check_file_access_at('utimensat')(debugger)
 
+    def do_fchmod(self, debugger: Debugger) -> bool:
+        path = self._getfd_pid(debugger.tid, debugger.uarg0)
+        return True if self.write_fs_jail.check(path) else ACCESS_EPERM(debugger)
+
+    def do_rename(self, debugger: Debugger) -> bool:
+        old_path, old_error = self.read_path('rename', debugger, debugger.uarg0)
```

Comment:
Nit: `old_path_error`? `old_error` and `new_error` sound weird.

Reply:
Fixed.